toolchain/driver/driver_fuzzer.cpp

```diff
@@ -78,7 +78,8 @@ extern "C" auto LLVMFuzzerTestOneInput(const unsigned char* data, size_t size)
     size -= arg_length;
   }
 
-  llvm::vfs::InMemoryFileSystem fs;
+  llvm::IntrusiveRefCntPtr<llvm::vfs::InMemoryFileSystem> fs =
+      new llvm::vfs::InMemoryFileSystem;
```

Comment:
In some cases you use `=` initialization, in others you use `()` like:

```
  llvm::IntrusiveRefCntPtr<llvm::vfs::InMemoryFileSystem> fs(
      new llvm::vfs::InMemoryFileSystem);
```

What system are you using to choose which style?

Reply:
Sorry, I was using `()` initialization for local variables, but that didn't work to default initialize fields. I've now switched the `()` cases to `=`. There are a lot of choices for initialization in C++, have we settled on a convention?